packages/7-todo-tdd/src/components/todo-app/TodoApp.test.tsx

```diff
@@ -1,157 +1,207 @@
-    })
-
-    it('should not add todo item with empty content', async () => {
-        appTestDriver.todoInputInsertContent('');
-        appTestDriver.clickOnAddBtn();
-        await appTestDriver.waitForAppToUpdate();
-        expect(appTestDriver.getTodosCount()).toBe(0);
-    });
-
-    it('should add todo item with content when pressing enter', async () => {
-        appTestDriver.todoInputInsertContent('example');
-        appTestDriver.pressEnterOnTodoInput();
-        await appTestDriver.waitForAppToUpdate()
-        expect(appTestDriver.getTodosCount()).toBe(1);
-    })
-
-    it('should add todo item with content when clicking on add btn', async () => {
-        appTestDriver.todoInputInsertContent('example');
-        appTestDriver.clickOnAddBtn();
-        await appTestDriver.waitForAppToUpdate()
-        expect(appTestDriver.getTodosCount()).toBe(1);
-    })
-
-    it('todo input should be empty after adding todo', async ()=>{
-        appTestDriver.todoInputInsertContent('example');
-        appTestDriver.clickOnAddBtn();
-        await appTestDriver.waitForAppToUpdate()
-        expect(appTestDriver.getTodoInputContent()).toBe(''); 
-    })
-})
+    expect(appTestDriver.getFirstTodo().content).toBe(content);
+  });
+
+  it('should add todo item with content when clicking on add btn', async () => {
+    const content = chance.word();
+    const { appTestDriver } = await givenTodos({});
+
+    expect(appTestDriver.getTodosCount()).toBe(0);
+
+    appTestDriver.todoInputInsertContent(content);
+    appTestDriver.clickOnAddBtn();
+    await appTestDriver.waitForAppToUpdate();
+
+    expect(appTestDriver.getFirstTodo().content).toBe(content);
+  });
+
+  it('todo input should be empty after adding todo', async () => {
+    const { appTestDriver } = await givenTodos({});
+
+    expect(appTestDriver.getTodoInputContent()).toBe('');
+
+    appTestDriver.todoInputInsertContent(chance.word());
+    appTestDriver.clickOnAddBtn();
+    await appTestDriver.waitForAppToUpdate();
+
+    expect(appTestDriver.getTodoInputContent()).toBe('');
+  });
+});
 
 describe('app should render fetched todos properly', () => {
-    let content: string;
-    let appTestDriver: TodoAppDriver;
-
-    beforeAll(async () => {
-        content = chance.word();
-        appTestDriver = new TodoAppDriver();
-        appTestDriver.todoInputInsertContent(content);
-        appTestDriver.clickOnAddBtn();
-        await appTestDriver.waitForAppToUpdate()
-    })
-
-    it('should render the correct _id', () => {
-        expect(Object.values(appTestDriver.getTodos())[0]).not.toBeUndefined()
-    })
-
-    it('should render the correct content', () => {
-        expect(Object.values(appTestDriver.getTodos())[0].content).toBe(content)
-    })
-
-    it('should render the correct state of isDone', () => {
-        expect(Object.values(appTestDriver.getTodos())[0].isDone).toBe(false)
-    })
-})
+  it('should render the correct _id', async () => {
```

Comment:
what does this test check?

Reply:
renamed it to "should render a valid UUID"

Reply:
renamed it to "should render a valid UUID"